description_sale_removal/models/product.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2024 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class Product(models.Model):
+    _inherit = "product.product"
+
+    def get_product_multiline_description_sale(self):
+        # Override to return only display_name
+        return self.display_name
```

Comment:
To leave an option to still consider description_sale in sales order line description proposal.  Complete override of a method is generally not advisable also.

```suggestion
    def get_product_multiline_description_sale(self):
        if not self.use_description_sale:
            return self.display_name
        return super().get_product_multiline_description_sale()
```

Add `use_description_sale` in product.template.

Please make adjustments to this suggestion so that it won't break standard tests.